### DISCUSSION TITLE
Suggestion: Adding vAmiga to the Homebrew Cask Repository

### DISCUSSION THREAD

**Discussion by sunfl0wer** [dirkwhoffmann/vAmiga#501] [Ideas]
Dear Prof. Dr. Hoffmann,
dear Developers,

first of all I would like to thank you for this great piece of software! Because vAmiga has left beta status a few days ago (congrats!) I would like to suggest to add it to the Homebrew Package Manager

https://brew.sh

and kindly ask for your permission to do so.

Regards,

Daniel

By the way: VirtualC64 is already present

https://formulae.brew.sh/cask/virtualc64#default

**Comment by dirkwhoffmann** [comment]
Hi Daniel,

I love the Homebrew Package Manager and would be very pleased to see the emulator included. I didn't know that VirtualC64 is already there. That's also great news!

**Reply by sunfl0wer** [reply]
I'm pretty sure you won't ever need to (because you're the lead developer ;-) but now you _could_ install vAmiga using Homebrew:

https://formulae.brew.sh/cask/vamiga#default

I hope this helps getting the popularity it deserves :-)

**Reply by dirkwhoffmann** [reply]
Excellent! Thanks a lot for adding the emulator.
